pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[tool.black]
+line-length = 100
```

Comment:
Boo!  80 characters or death! :stuck_out_tongue:

Reply:
:) just basing it off of our existing config